viam-cartographer/src/slam_service/slam_service.h

```diff
@@ -47,24 +47,32 @@ namespace viam {
 
 static const int checkForShutdownIntervalMicroseconds = 1e5;
 
-// This RGB value represents "no input" from a Cario pained map
-static const unsigned char defaultCairosEmptyPaintedSlice = 102;
 static const int jpegQuality = 50;
 // Byte limit on GRPC, used to help determine sampling skip_count
 static const int maximumGRPCByteLimit = 32 * 1024 * 1024;
 // Byte limit for chunks on GRPC, used for streaming apis
-static const int maximumGRPCByteChunkSize = 64 * 1024;
-// Coeffient to adjust the skip count for the PCD to ensure the file is within
-// grpc limitations. Increase the value if you expect dense feature-rich maps
-static const int samplingFactor = 1;
-// Conversion to number of bytes used in colored PCD encoding
-static const int pixelBytetoPCDByte = 16 / 4;
+static const int maximumGRPCByteChunkSize = 1 * 1024 * 1024;
+
 // Quaternion to rotate axes to the XZ plane
 static const Eigen::Quaterniond pcdRotation(0.7071068, -0.7071068, 0, 0);
 // Static offset quaternion, so orientation matches physical intuition.
 // This will result in rotations occurring within the y axis to match 2D mapping
 // in the XZ plane
 static const Eigen::Quaterniond pcdOffsetRotation(0.7071068, 0.7071068, 0, 0);
+
+// The resolution_meters variable defines the area in meters that each pixel
+// represents. This is used to draw the jpeg map and in so doing defines the
+// resolution of the outputted PCD
+const double resolution_meters = 0.05;
```

Comment:
Discussed offline. Delaying a decision on this until we expose this to the user. For now, also keeping it as meters because that is cartographer's scale/